tflite_micro_compiler/src/Compiler.cc

```diff
@@ -456,6 +456,7 @@ void tflmc::Compiler::writeSource(std::ostream &out) {
 #include "tensorflow/lite/micro/micro_context.h"
 
 // #define TFLMC_XCORE_PROFILE
+// #define TFLMC_CONV2D_PROFILE
```

Comment:
If we only enable TFLMC_CONV2D_PROFILE without enabling TFLMC_XCORE_PROFILE, does it work?